runtime/README.md

```diff
@@ -1,7 +1,22 @@
-# Caliptra Runtime Firmware v1.1
+# Caliptra Runtime Firmware v2.0
 
 This specification describes the Caliptra Runtime Firmware.
 
+## New in v1.1
+
+* [LMS Signature Verification](#lms_signature_verify-11)
+
+## New in v1.2
+
+* [Manifest-Based Image Authorization](#manifest-based-image-authorization-12)
+
+## New in v2.0
+
+Caliptra 2.0 firmware supports passive mode (same as 1.x) and active (or susbystem) mode.
```

Comment:
suggest replacing active with subsystem throughout the document to parallel the main spec. subsystem is misspelled here.